docs/defining_guards/pydantic.ipynb

```diff
@@ -80,8 +80,8 @@
     "guard = Guard.from_pydantic(output_class=Pet, prompt=prompt)\n",
     "\n",
     "raw_llm_output, validated_output, *rest = guard(\n",
-    "    llm_api=openai.completions.create,\n",
-    "    engine=\"text-davinci-003\"\n",
+    "    llm_api=openai.chat.completions.create,\n",
```

Comment:
Is there a reason to switch from using completions to chat completions?  Is it better at instruction following this way?